GOVERNANCE.md

```diff
@@ -0,0 +1,62 @@
+# Scope Governance
+
+This document defines project governance for the project. This is (and probably will always be) a Work in Progress.
+
+## Goals and Principles
+
+Scope's community goals and principles:
+
+1. Transition from a primarily Weaveworks project to a true community-driven project with autonomous governance.
+2. Fill-in the needs of the community with a chop-wood-and-carry-water attitude: expect to give back before you take if you want to make an impact. Demands and suggestions from community members will be taken into account but actions and help will be more-highly appreciated.
+
+[Alfonso Acosta](https://github.com/2opremio) will be in charge of ramping up the community: establishing governance, initial maintenance and chairing. Once the community reaches critical mass and sufficient maintainers are designed, the community will be self-managed.
```

Comment:
Scope's community is already benefiting from your work and you should be applauded. I just worry a bit that a sentence like the above might put you into a situation where people will feel like "ah, it's a community thing: fons will take care of it" and you will deprive yourself of additional scope community captains. :-)

Reply:
Happy to remove it (that thought ran through my head when I incorporated that as well).